src/helpers/queries.js

```diff
@@ -0,0 +1,34 @@
+import { Q, fetchPolicies } from 'cozy-client'
+import { DOCTYPE_CONTACTS } from './doctypes'
+
+export const cacheTime = 30000
```

Comment:
const olderThan30s = fetchPolicies.olderThan(30000)
...

fetchPolicy: olderThan30s

1/ more readable 
2/ only one assignation (not twice in every render)